opm/grid/cpgpreprocess/preprocess.c

```diff
@@ -80,6 +81,13 @@ linearindex(const int dims[3], int i, int j, int k)
     return i + dims[0]*(j + dims[1]*k);
 }
 
+static int
+vertical_cart_neighbors(const int dims[3], int i, int j){
```

Comment:
Could you use names other than `i` and `j` here please?  This file generally uses those names for the `I` and `J` Cartesian indices of a cell (or a vertex).

Reply:
Will do.